android/app/src/main/kotlin/com/emergetools/hackernews/ui/items/Comment.kt

```diff
@@ -35,6 +36,7 @@ fun BuildComment(
  * Example generated snapshot test from main source set.
  * To generate a snapshot test for this preview, add the snapshot-processor as a ksp dependency.
  */
+@SnapshotTestingPreviews
 @Preview
```

Comment:
I'd say just remove the stock @Preview now that we have a multipreview.

Reply:
Figured I'd leave it in since it's yet another edge case to consider 😅